scipy/stats/_morestats.py

```diff
@@ -4057,10 +4057,10 @@ def directional_stats(samples, *, axis=0, normalize=True):
 
     Returns
     -------
-    directional_mean : ndarray
+    mean_resultant : ndarray
         Directional mean.
-    magnitude : ndarray
-        Magnitude of the directional mean. This can be used to calculate
+    mean_resultant_magnitude : ndarray
```

Comment:
Should `mean_resultant_magnitude` this not be float?

Reply:
It could be a `ndarray` when multiple inputs are stacked. Example: 

```ipython
In [1]: import numpy as np

In [2]: from scipy import stats

In [3]: samples = np.random.randn(2, 5, 3)

In [4]: stats.directional_stats(samples).mean_resultant_length.shape
Out[4]: (5,)
```

Reply:
Also, as I mentioned in the previous comment, I named it `mean_resultant_magnitude` by mistake. It has been changed to `mean_resultant_length` now as you suggested before.